From 3f5f6ac49af273b578999f0a3285e9fab5a9156c Mon Sep 17 00:00:00 2001
From: Sushant Daspute <sushant_daspute@comcast.com>
Date: Tue, 15 Mar 2022 11:46:14 +0000
Subject: [PATCH] DELIA-54629 : Unable to share the screen only 500 Server
 Error

Test Procedure: Check snmpv3 working with xray
Reason for change: disabling additional signature algorithms found on
server hello with openssl1.1.1
Risks: Low

Signed-off-by: Sushant Daspute <sushant_daspute@comcast.com>
---
 ssl/t1_lib.c | 9 ---------
 1 file changed, 9 deletions(-)
Index: openssl-1.1.1g/ssl/t1_lib.c
===================================================================
--- openssl-1.1.1g.orig/ssl/t1_lib.c
+++ openssl-1.1.1g/ssl/t1_lib.c
@@ -640,17 +640,8 @@ static const uint16_t tls12_sigalgs[] =
     TLSEXT_SIGALG_ecdsa_secp256r1_sha256,
     TLSEXT_SIGALG_ecdsa_secp384r1_sha384,
     TLSEXT_SIGALG_ecdsa_secp521r1_sha512,
-    TLSEXT_SIGALG_ed25519,
-    TLSEXT_SIGALG_ed448,
 #endif
 
-    TLSEXT_SIGALG_rsa_pss_pss_sha256,
-    TLSEXT_SIGALG_rsa_pss_pss_sha384,
-    TLSEXT_SIGALG_rsa_pss_pss_sha512,
-    TLSEXT_SIGALG_rsa_pss_rsae_sha256,
-    TLSEXT_SIGALG_rsa_pss_rsae_sha384,
-    TLSEXT_SIGALG_rsa_pss_rsae_sha512,
-
     TLSEXT_SIGALG_rsa_pkcs1_sha256,
     TLSEXT_SIGALG_rsa_pkcs1_sha384,
     TLSEXT_SIGALG_rsa_pkcs1_sha512,
